data/test-cases/plugins/zoom/general/Meeting_in_threads.md

```diff
@@ -0,0 +1,57 @@
+---
+# (Required) Ensure all values are filled up
+name: "Create a Zoom meeting in the threads on MM."
+status: Active
+priority: Normal
+folder: General
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Connect the Zoom account to you MM account.
+2. Enable the CRT(if disabled) in the settings on MM and open the `threads` from the LHS on MM.
```

Comment:
Plugin config, instead of settings

Reply:
I think i should refactor it to system console, as the CRT is not part of the plugin config but for the posts settings in the System console.